packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -49,17 +56,38 @@ export interface ArticleMetadataPanelProps {
 }
 
 export function ArticleMetadataPanel({value, onClose, onChange}: ArticleMetadataPanelProps) {
-  const {preTitle, title, lead, tags, authors, shared, breaking, image, hideAuthor} = value
+  const {
+    preTitle,
+    title,
+    lead,
+    tags,
+    authors,
+    shared,
+    breaking,
+    image,
+    hideAuthor,
+    socialMediaTitle,
+    socialMediaDescription,
+    socialMediaAuthors,
+    socialMediaImage
+  } = value
+
+  const [activeKey, setActiveKey] = useState(MetatagType.General)
 
   const [isChooseModalOpen, setChooseModalOpen] = useState(false)
   const [isEditModalOpen, setEditModalOpen] = useState(false)
+  const [chosenImageType, setChosenImageType] = useState('')
```

Comment:
why don't you set the image id here instead of which type.

Reply:
currentImage

Reply:
done